frontend/router.ts

```diff
@@ -58,31 +62,27 @@ class Router extends Observable {
       this.year = year
       this.month = month
     }
-    this.commitDateChange()
-    if (path === '') {
-      if (store.id) {
-        this.go('list')
-        return
-      }
-      this.go('login')
+    if (!store.token) {
+      this.go(ROUTE.LOGIN)
       return
     }
+    if (path === '') this.go(ROUTE.LIST)
     if (!this.isInvalidPath(path)) this.go(path)
```

Comment:
```suggestion
if (this.isInvalidPath(path)) return
 this.go(path)
```
이게 더 직관적으로 보일 것 같습니다 😎